packages/vscode/syntaxes/astro.tmLanguage.json

```diff
@@ -131,6 +170,61 @@
         }
       },
       "patterns": [
+        {
+          "comment": "Treat script tag with application/ld+json as JSON. This needs to be higher than is:raw since it's a very possible situation to have is:raw on a JSON script tag",
+          "begin": "\\G(?=\\s*[^>]*?type\\s*=\\s*(['\"]|)(?i:application/ld\\+json)\\1)",
+          "end": "(?=</|/>)",
+          "patterns": [
+            {
+              "begin": "(?<=>)(?!</)",
+              "end": "(?=</)",
+              "contentName": "source.json",
+              "patterns": [{ "include": "source.json" }]
+            },
+            { "include": "#html:tag-attributes" }
+          ]
+        },
+        {
+          "comment": "Treat script tag with is:raw as plain text",
```

Comment:
Curious about the highlighting here... I don't think `is:raw` has any effect on `script` or `style` (braces are already ignored), so do we really want highlighting to change to plaintext? I think everything will still be parsed and bundled.

Reply:
Ah, I might have misunderstood what `is:raw` does on those tags then! 

I thought it would act like a more intense `is:inline` basically. Especially for script tags, I felt like having a weird type + `is:raw` could be a "common" case

Reply:
Decided to remove `is:raw` as plaintext for style tags and script tags after confirming that it doesn't do anything in Astro